swagger/v1/person_escort_record.yaml

```diff
@@ -0,0 +1,62 @@
+PersonEscortRecord:
+  type: object
+  required:
+    - id
+    - type
+    - attributes
+  properties:
+    id:
+      type: string
+      format: uuid
+      example: f0a91e16-1b0e-4e1f-93fe-319dda9933e6
+      description: The unique identifier (UUID) of this object
+    type:
+      type: string
+      example: person_escort_records
+      enum:
+      - person_escort_records
+      description: The type of this object - always `person_escort_records`
+    attributes:
+      type: object
+      required:
+      - version
+      - status
```

Comment:
Hmm.. do we really want the POST PER endpoint to require status to be sent? We've moved away from that pattern with moves and now have explicit event endpoints to change status. Maybe the status should be read only?